File: JudgingMoose/judgingMoose.py
def main():
    l, r = map(int, input().split())
    #res = 'Even ' + str(l+r) if l == r else 'Odd ' + str(2*max(l,r))
    
    if l == 0 and r == 0:
        print('Not a moose')
        return
    
    if l == r:
        print('Even ' + str(l+r))
    else:
        print('Odd ' + str(2*max(l,r)))
    
main()

File: JudgingMoose/judgingMoose.cpp
//https://open.kattis.com/problems/judgingmoose
#include<iostream>

using namespace std;

int main() {
    
    int l, r;
    cin >> l >> r;
    
    if(l <= 0 && r <= 0) {
        cout << "Not a moose" << endl;
        return 0;
    }
    
    if(l == r) {
        cout << "Even " << l+r << endl;
    } else {
        cout << "Odd " << 2*max(l,r) << endl;
    }
    
    return 0;
}